chromadb/test/test_api.py

```diff
@@ -676,6 +676,53 @@ def test_where_ne_eq_number(api):
     assert len(items["metadatas"]) == 1
 
 
+def test_where_like_string(api):
```

Comment:
Does it make sense to also add character level match with `_`? Also it would be nice to test escapes `\%`, `\_`

Reply:
Let's also test for partial and exact matches (see comment below)

Reply:
Can we also add a test for case-sensitivity. Chroma enables case-sensitivity (although it is buggy for now but there is a PR to fix it) with `cur.execute("PRAGMA case_sensitive_like = ON")`

Reply:
added a separate test for case sensitivity, additional tests for the `_` wildcard character, and escaped wildcard characters `\%`, `\_` in the commit.